packages/widget/src/hooks/useAvailableChains.ts

```diff
@@ -22,7 +23,7 @@ export const useAvailableChains = (chainTypes?: ChainType[]) => {
   // const { providers } = useHasExternalWalletProvider();
   const { data, isLoading } = useQuery({
     queryKey: [
-      'chains',
+      `${lifiWidgetQueryPrefix}-chains`,
```

Comment:
We have a customizable `keyPrefix` in the widget config, which should be taken into account here and other similar places I guess.